scripts/remove-old-image.py

```diff
@@ -0,0 +1,82 @@
+"""
+
+This script is based on this documentation: 
+https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28
+
+"""
+
+import os
+import requests
+from requests.auth import HTTPBasicAuth
+
+# Delete the old container (based on the previous tag)
+def delete_old_image(version_id, org, headers, auth):
+    url_delete_previous_version = f"https://api.github.com/orgs/{org}/packages/container/{container_name}/versions/{version_id}"
+    response = requests.delete(url_delete_previous_version, headers=headers, auth=auth)
+    if response.status_code == 204:
+        print(f'Previous container deleted!')
+    else:
+        print('error deleting the previous container:', response.status_code, response.text)
+        exit(1)
```

Comment:
don't call exit, raise exception. call exit in top-level function

Reply:
Done. Check this pull request https://github.com/ai-cfia/devops/pull/27